### DISCUSSION TITLE
rbenv users?

### DISCUSSION THREAD

**Discussion by huyz** [rbenv/rbenv#1525] [Q&A] [answered]
Is there an equivalent command, app, or technique for something like https://github.com/pyenv/pyenv-users ?

**Comment by mislav** [comment, marked as answer]
I'm not aware of such a plugin existing, but it's a cool use case!

Here's a one-liner that could answer a question about which projects within the `~/Projects` directory currently uses the `2.7.0` Ruby version:
```
find ~/Projects -name '.ruby-version' -exec grep -qF 2.7.0 '{}' \; -print
```
Explanation:
- `find` searches for files named `.ruby-version` under a given starting path (`~/Projects`);
- for each found file, match its contents against the literal string `2.7.0`;
- the `{}` and `\;` bits might look confusing, but are explained in `man find` under the `-exec` section;
- print all matches.

**Reply by huyz** [reply]
Thanks for that.  I was looking for something more comprehensive that lists all the versions used.  But knowing that no analogue to pyenv-users exists is mostly what I wanted to know.
